packages/base/src/commands.ts

```diff
@@ -293,33 +315,18 @@ export function addCommands(
   commands.addCommand(CommandIDs.buffer, {
     label: trans.__('Buffer'),
     isEnabled: () => {
-      const model = tracker.currentWidget?.model;
-      const localState = model?.sharedModel.awareness.getLocalState();
-
-      if (!model || !localState || !localState['selected']?.value) {
+      const selectedLayer = getSelectedLayer(tracker);
+      if (!selectedLayer) {
         return false;
       }
-
-      const selectedLayers = localState['selected'].value;
-
-      if (Object.keys(selectedLayers).length > 1) {
-        return false;
-      }
```

Comment:
We don't have this logic anymore. How about putting this logic in `getSelectedLayer` and maybe rename `getSelectedLayer` into `getSingleSelectedLayer` (or something else that makes more sense if you have ideas)?